app/assets/stylesheets/styles/components/pages/_home.css.scss

```diff
@@ -178,24 +128,16 @@ video { display: block; }
     }
 
     input.search-button {
+      background-image: none;
+      background-color: $of-green;
+      border-radius: 0;
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.